db/migrate/20160829220123_create_logs.rb

```diff
@@ -0,0 +1,23 @@
+class CreateLogs < ActiveRecord::Migration
+  def up
+    execute(<<-SQL
+      CREATE TABLE "logs" (
+        "id" INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+        "user_id" INTEGER,
+        "customer_id" INTEGER,
+        "summary" TEXT,
+        "duration" INTEGER,
+        "created_at" DATETIME,
+        "updated_at" DATETIME
+      )
+      SQL
+    )
+
+    execute('CREATE INDEX "index_logs_on_customer_id" ON "logs" ("customer_id")')
+    execute('CREATE INDEX "index_logs_on_user_id_and_created_at" ON "logs" ("user_id", "created_at")')
```

Comment:
Add a `created_at` index (this one won't cover queries that don't use `user_id`).